src/components/Login.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container fluid fill-height class="pa-0 ma-0">
+    <v-layout align-center justify-center>
+      <v-flex text-xs-center fill-height>
+        <v-card dark tile
+                color="#36363f"
+                height="100%"
+          >
+          <v-toolbar dark flat color="#474755">
+            <v-toolbar-title> Login </v-toolbar-title>
+          </v-toolbar>
+          <v-icon size="196px" class="pt-5"> mdi-account-circle</v-icon>
+          <div :class="{'margin-large': $vuetify.breakpoint.lgAndUp,
+                        'margin-medium': $vuetify.breakpoint.md,
+                        'margin-small': $vuetify.breakpoint.smAndDown,}">
+            <v-text-field v-model="user.email"
+                          label="Email"
+                          name="email"
+                          light
+                          required
+                          solo
+            ></v-text-field>
+            <v-text-field v-model="user.password"
+                          label="Password"
+                          name="password"
+                          type="password"
+                          light
+                          required
+                          solo
+            ></v-text-field>
+          </div>
+          <v-btn light large round class="ma-2">Login</v-btn><br>
```

Comment:
We are following flt design, remove all shadows

Reply:
flat***. This also includes the text fields

Reply:
done